command/certificate/format.go

```diff
@@ -21,34 +21,34 @@ func formatCommand() cli.Command {
 		Action:    command.ActionFunc(formatAction),
 		Usage:     `reformat certificate`,
 		UsageText: `**step certificate format** <crt_file> [**--out**=<file>]`,
-		Description: `**step certificate format** prints the certificate in
-a different format.
+		Description: `**step certificate format** prints the certificate or certificate signing
+request in a different format.
```

Comment:
nitpicking, but I would use `csr` here since that's what you use in the documentation below.

Reply:
Fixed in ad3e20a